integrations/spring-boot/src/it/multi-manager/src/main/java/test/microstream/integrations/spring/boot/RootPreparationOfGreenDatabase.java

```diff
@@ -0,0 +1,62 @@
+package test.microstream.integrations.spring.boot;
+
+/*-
+ * #%L
+ * microstream-integrations-spring-boot
+ * %%
+ * Copyright (C) 2019 - 2022 MicroStream Software
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import one.microstream.integrations.spring.boot.types.config.StorageManagerInitializer;
+import one.microstream.storage.types.StorageManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import test.microstream.integrations.spring.boot.database.DatabaseColor;
+import test.microstream.integrations.spring.boot.database.Names;
+
+@Component
+public class RootPreparationOfGreenDatabase implements StorageManagerInitializer {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RootPreparationOfGreenDatabase.class);
+
+    @Override
+    public void initialize(final StorageManager storageManager) {
+        if (!DatabaseColor.GREEN.getName().equals(storageManager.databaseName())) {
```

Comment:
Isn't there any way to solve this through the qualifier?

Reply:
I get a list of all beans implementing StorageManagerInitializer from Spring. I can have a look at the Class definition and look to see if there is a Qualifier or not and it that matches to know if I need to call the initialize method for that StorageManager.

I'll have a look to implement that solution also.